Spec/RestClient.swift

```diff
@@ -173,8 +173,20 @@ class RestClient: QuickSpec {
                     
                     expect(testHTTPExecutor.requests.first?.URL?.host).toEventually(equal("fake.ably.io"), timeout: testTimeout)
                 }
+
+                it("should ignore an environment when host is not in production") {
+                    let options = ARTClientOptions(key: "fake:key")
+                    options.environment = "test"
+                    options.restHost = "fake.ably.io"
+                    let client = ARTRest(options: options)
+                    client.httpExecutor = testHTTPExecutor
+
+                    publishTestMessage(client, failOnError: false)
+
+                    expect(testHTTPExecutor.requests.first?.URL?.host).toEventually(equal("fake.ably.io"), timeout: testTimeout)
```

Comment:
OK, [TOK3k2](http://docs.ably.io/client-lib-development-guide/features/#TO3k2) clarifies it.  It is undefined because it is invalid. I guess, for lack of a more specific meaning for "invalid", that just ignoring the environment here is fine then.